lib/inspec/cli.rb

```diff
@@ -175,14 +175,19 @@ def archive(path)
   end
 
   desc 'exec LOCATIONS', 'run all test files at the specified LOCATIONS.'
+  # TODO: find a way for Thor not to butcher the formatting of this
```

Comment:
It's not looking so good rendered either:

![2019-01-15-225555_713x391_scrot](https://user-images.githubusercontent.com/13783510/51231560-c82fdc00-1918-11e9-8caf-00e9a560c795.png)

This gets us closer, but Thor still doesn't like it.

```diff
    Exit codes:
      ```
      0:   Normal exit, all tests passed
      1:   Usage or general error
      2:   Error in plugin system
      3:   Fatal deprecation encountered
      100: Normal exit, at least one test failed
      101: Normal exit, at least one test skipped but none failed
      ```
```

![2019-01-15-230358_714x503_scrot](https://user-images.githubusercontent.com/13783510/51231876-db8f7700-1919-11e9-857f-40971a969717.png)

Maybe we should ask @kagarmoe?